amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import ReactDOM from 'react-dom';
+import ReactMarkdown from 'react-markdown';
```

Comment:
And both `import * as ReactMarkdown from 'react-markdown';` and `import { ReactMarkdown } from 'react-markdown';` fail as well without this option? It's very odd that such a popular library wouldn't be compatible with ES6 imports without that option.

Reply:
First one helped, rolling back tsconfig changes